_layouts/default.html

```diff
@@ -29,6 +29,9 @@
     {% endif %}
 </head>
 <body class="{{page.layout}}-layout">
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KLNNSNV" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
Shouldn't id be configurable (Same for JS in head.html)? Otherwise, if customers will add those snippets all the requests will go to our account..

Reply:
Not sure, why should customers add this script somewhere?